_layouts/default.html

```diff
@@ -13,7 +13,9 @@
 
     <header>
       <div class="container">
-        <h1>{{ site.title | default: site.github.repository_name }}</h1>
+        <a href="/" style="text-decoration: none;">
```

Comment:
Two suggestions here:

1. For discoverability, I think the style should go in the stylesheet, rather than in-line
2. We'll need to use `{{ "/" | relative_url }}` to account for sites in subdirectories

Reply:
Style at stylesheet and `{{ "/" | relative_url }}` in next commit